src/bentoml/_internal/yatai_rest_api_client/yatai.py

```diff
@@ -387,3 +388,18 @@ def download_model(
         resp = self.session.get(url, stream=True)
         self._check_resp(resp)
         return resp
+
+    def get_latest_model(
```

Comment:
I feel like since the bento cloud client PR is going to get merged soon, we should also include these in there.

Reply:
No problem, I can resolve the conflicts after it is merged.

Reply:
sg if that is not too much of a hassle for you ❤️